scripts/static-linking.patch

```diff
@@ -0,0 +1,118 @@
+From 25b42b96591d7a682b5884160b54b5da8dd3036d Mon Sep 17 00:00:00 2001
+From: Christoph Knittel <ck@cca.io>
+Date: Sat, 26 Nov 2022 10:04:01 +0100
+Subject: [PATCH] Static linking
+
+---
+ jscomp/bsb_exe/dune        | 3 ++-
+ jscomp/bsb_helper_exe/dune | 3 ++-
+ jscomp/bsc/dune            | 3 ++-
+ jscomp/cmij/dune           | 3 ++-
+ jscomp/ounit_tests/dune    | 3 ++-
+ res_syntax/benchmarks/dune | 3 ++-
+ res_syntax/cli/dune        | 3 ++-
+ res_syntax/testrunner/dune | 3 ++-
+ 8 files changed, 16 insertions(+), 8 deletions(-)
+
+diff --git a/jscomp/bsb_exe/dune b/jscomp/bsb_exe/dune
+index e75592a47..369a1c1fa 100644
+--- a/jscomp/bsb_exe/dune
++++ b/jscomp/bsb_exe/dune
+@@ -2,5 +2,6 @@
+  (name rescript_main)
+  (public_name rescript)
+  (flags
+-  (:standard -w +a-4-9-40-41-42-70))
++  (:standard -w +a-4-9-40-41-42-70)
++  (-cclib -static -cclib -no-pie))
```

Comment:
Is there no way in dune to make this configurable, so that a patch is the only solution?

Reply:
I did not find a way to make it configurable.

Reply:
how about using the `env` stanza and compiling with the `--profile` flag? e.g. https://github.com/anmonteiro/piaf/blob/master/dune#L4

Reply:
@anmonteiro Happy New Year and thanks a lot for the hint! 😄

Reply:
See #5922.